### PR TITLE
{vis}[intel/2015a] libXp 1.0.2 + deps (REVIEW)

### PR DESCRIPTION
**Pull request by boegel**
equivalent of #1321 for `intel/2015a`, to help resolve a dependency hell I ran into...

no new easyconfigs, only `intel/2015a` versions of existing `intel/2014b` easyconfigs

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 15 out of 15 (13 easyconfigs in this PR)
Linux SL 6.5, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/77672e5eb0df209fe7ee for a full test report.

**Comment by boegel** [comment]
@wpoely86: please review?

**Comment by wpoely86** [comment]
well, if only the toolchain is different from #1321, I have no more remarks.

**Comment by boegel** [comment]
on hold until `libxcb` build issues with `Autoconf`/`Automake` as build deps in #1321 can be resolved...

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 6 out of 6 (6 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/289ff7880e0c74d7251a for a full test report.

**Comment by boegel** [comment]
@wpoely86: please rereview?

File: easybuild/easyconfigs/l/libXp/libXp-1.0.3-intel-2015a.eb [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+builddependencies = [
+     ('xextproto', '7.3.0'),
+     ('printproto', '1.0.5'),
+]
+
+dependencies = [
+     ('libX11', '1.6.3', '-Python-2.7.9'),
+     ('libXext', '1.3.3'),
+     ('libXau', '1.0.8'),
+]
```
Comment (wpoely86) [inline easybuild/easyconfigs/l/libXp/libXp-1.0.3-intel-2015a.eb]: add newline below

File: easybuild/easyconfigs/l/libX11/libX11-1.6.3-intel-2015a-Python-2.7.9.eb [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+homepage = "http://www.freedesktop.org/wiki/Software/xlibs"
+description = """X11 client-side library"""
+
+toolchain = {'name': 'intel', 'version': '2015a'} 
+
+sources = [SOURCE_TAR_GZ]
+source_urls = [XORG_LIB_SOURCE]
+
+builddependencies = [
```
Comment (wpoely86) [inline easybuild/easyconfigs/l/libX11/libX11-1.6.3-intel-2015a-Python-2.7.9.eb]: in all the others, deps are first, then builddeps.

Comment (boegel) [inline easybuild/easyconfigs/l/libX11/libX11-1.6.3-intel-2015a-Python-2.7.9.eb]: actually only in one of them, and we generally include build deps first, so I'll fix the other one

File: easybuild/easyconfigs/l/libXp/libXp-1.0.3-intel-2015a.eb (hunk 1 above)
Comment (boegel) [inline easybuild/easyconfigs/l/libXp/libXp-1.0.3-intel-2015a.eb]: done

**Comment by boegel** [comment]
good to go in, synced with #1321, thanks for the review @wpoely86!

**Pull request merged**